.github/dependabot.yml

```diff
@@ -0,0 +1,139 @@
+version: 2
+updates:
+  - package-ecosystem: "gradle"
+    directory: "/packages/flutter_plugin_android_lifecycle/example/android/app"
```

Comment:
Will checking the examples transitively check and update the plugin dependencies themselves?

Reply:
I don't think so. It's not very smart from what I can tell